client/verta/verta/environment/_python.py

```diff
@@ -203,11 +219,29 @@ def _capture_constraints(self, constraints):
                             " not {}".format(type(constraints)))
 
         self._msg.python.constraints.extend(
-            self._req_spec_to_msg(req_spec)
-            for req_spec
-            in req_specs
+            map(
+                self._req_spec_to_msg,
+                req_specs,
+            ),
         )
 
+    @staticmethod
+    def _remove_torch_metadata(req_specs):
```

Comment:
We should not assume this is only an issue for pytorch, as I'd rather not have another case come up. Is `+` in the install name valid according to pip? If not, we should just remove from all.

Reply:
sg. let's just have torch for now then

Reply:
While looking at robustifying the test, I found this in the PEP:
> Local version identifiers SHOULD NOT be used when publishing upstream projects to a public index server . . .

So it now makes more sense to remove this for all packages, here.